data/use_cases.yml

```
@@ -3,19 +3,39 @@
 developer_onboarding:
   title: Developer Onboarding
   link: /strategy-goals/strategy/use_cases/dev-onboarding
+  sponsors:
+    product: ben_venker
+    engineering: jason_yavorska
+    design: jason_yavorska
 
 code_reuse:
   title: Code Reuse
   link: /strategy-goals/strategy/use_cases/code_reuse
+  sponsors:
+    product: jason_yavorska
+    engineering: jason_yavorska
```

Happy to do this as well if nobody else comes along!

```suggestion
    engineering: robert_lin
```